netci.groovy

```diff
@@ -436,7 +436,7 @@ def static addCopyCoreClrAndRunTestSteps(def job, def coreclrBranch, String os,
             }
 
             // Unpack the build data
-            shell("unpacker ./bin/build.pack ./bin")
+            shell("unpacker ./bin/build.pack .")
             // Export the LTTNG environment variable and then run the tests
```

Comment:
We unpack to the root because with this change we are packing up both the bin and packages folders on Windows.  So we need to unpack to the root because we want the packages folder to be in the root.

Previously we unpacked into bin because the packing on Windows was just done under bin.

Run-Test.sh and cmd will be updated to just call the script that build generates which deploys test assets and invokes xUnit.  Because of that, we need to pass the relevant options to the build script so the test runner script it generates is correct.